core/src/main/java/de/erethon/commons/misc/BlockUtil.java

```diff
@@ -13,15 +13,18 @@
 package de.erethon.commons.misc;
 
 import de.erethon.commons.compatibility.Version;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import org.bukkit.Location;
 import org.bukkit.block.Block;
 import org.bukkit.block.BlockFace;
-import static org.bukkit.block.BlockFace.*;
+import org.bukkit.block.Sign;
+import org.bukkit.block.data.type.WallSign;
 import org.bukkit.material.Attachable;
 import org.bukkit.material.MaterialData;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.*;
```

Comment:
https://youtu.be/H_pp130gYGQ?t=14

Es wäre schön, wenn Du es so formatieren könntest, wie es überall sonst ist (ohne Lücken, keine Stars). Star-Imports sollten benutzt werden, wenn wirklich _jede_ Klasse gemeint ist, nicht als Wildcard für irgendwelche Klassen im Package.

Reply:
War gar nicht gewollt, macht IntelliJ leider automatisch 😄